build.ps1

```diff
@@ -364,18 +362,20 @@ Remove-Item variable:global:_MSBuildExe -ea Ignore
 # Import Arcade
 . "$PSScriptRoot/eng/common/tools.ps1"
 
+# Add default .binlog location if not already on the command line. tools.ps1 does not handle this; it just checks
+# $BinaryLog, $CI and $ExcludeCIBinarylog values for an error case. But tools.ps1 provides a nice function to help.
+if ($BinaryLog) {
+    $bl = GetMSBuildBinaryLogCommandLineArgument($MSBuildArguments)
```

Comment:
This and the similar `for` loop in `build.sh` are somewhat better than was here before. The scripts won't bother to add a `/bl:Build.binlog` option if anything one starting with `/bl:` or `/binaryLogger:` (plus `-bl:` or `-binaryLogger:` in the build.sh case) already exists.